### ISSUE TITLE
Conflict between Favorites v1.3.0 and Note Tabs v1.4.0 plugins

### ISSUE THREAD

**Issue by nickr812** [laurent22/joplin#7584]
Using Joplin on Windows v2.9.17 I have both Favorites v1.3.0 and Note Tabs v1.4.0 plugins installed and active.

With this configuration it is impossible to toggle Tabs visibility to On.

If you select this and then select View / Change Application Layout you can choose where the Tabs pane should go, but when you exit this screen the tabs remain hidden.

If you then deactivate the Favorites plugin and restart the Tabs behave as expected.

## Environment

Joplin version: v2.9.17
Platform: Windows 11 Home Build 22621.963
OS specifics: 

## Steps to reproduce

1. As above
2. 
3.

**Comment by Divakar-Chouhan** [comment]
The tab section isn't visible even after deactivating the favorites plugin

https://user-images.githubusercontent.com/87907610/214828141-0372bb27-4fbe-425a-903b-2839480218ae.mp4

**Comment by pedr** [comment]
I'm not sure if understand the problem. 

I tried to find some unexpected behavior with these plugins installed, toggling the visibility of elements and their configuration but I couldn't really find anything that seemed a bug for me. I tried to replicate it on Ubuntu, later I will be able to test it on Windows too.

I think what might cause some confusion is the fact that when you enter the Change application layout mode the elements that have the visibility toggled off are also shown in the grid, I think it is understandable why it might cause some confusion. I'm not sure if this is an unexpected behavior or an intentional choice @laurent22 

For example, if the Note Tabs are hidden I was able to toggle the visibility from the menu:
![image](https://user-images.githubusercontent.com/5051088/220144130-45636f45-09d8-479a-a07e-7d4bb02cee69.png)

Or if the Note list is hidden:
![image](https://user-images.githubusercontent.com/5051088/220146945-10e2886b-9b22-413c-96c0-22fda38eb260.png)
